keyboard.html

```diff
@@ -40,18 +40,20 @@
       const keysPerOctave = keyNames.length
       let i = 0;
       let keyCount = 0;
+      let octaveCount = 2 //start at the second octave;
```

Comment:
why start a 2nd octive?

Reply:
to make it more dynamic you might want to allow `addKeys` to take in a second param `function addKeys(numKeys, startingOctive) {}`

Reply:
not really necessary to start at 2nd octave right now, but went ahead and make a note since we likely want to keep our melodies between the 2nd and 4th octaves since they'll be more pleasing to the ear